tasks/containers/TaskWorker.js

```diff
@@ -79,14 +79,32 @@ export default compose(
 
```

Comment:
which part always returns true @NotThatSmith ? if you're online in the next 40 mins or so i can take a look at it with you